drizzlepac/util.py

```diff
@@ -269,24 +274,24 @@ def output(msg):
     for software in pkgs:
         try:
             package = __import__(software)
-            vstr = "%s "%(software)
+            vstr = "%s " % (software)
             try:
-                vstr+= "Version -> "+package.__version__+" "
-            except:
+                vstr += "Version -> " + package.__version__ + " "
+            except Exception:
                 vstr += "No version defined.  "
             if svn:
                 git = True
-                #try:
+                # try:
                 #    vstr += "\n    SVN version -> "+package.__svn_version__.rstrip()
-                #except:
+                # except:
                 #    vstr += " "
```

Comment:
Does the SVN block need to exist?  If so, is just git = True sufficient?

Reply:
This section deleted in b11537b4